aydin/nn/models/unet.py

```diff
@@ -139,6 +138,13 @@ def __init__(
         self.compile(optimizer=Adam(lr=learning_rate), loss='mse')
         self.compile = True
 
+    def need_batch_size_one(self):
+        return 'shiftconv' in self.training_architecture
+
+    def size(self):
+        """Returns size of the model in bytes"""
+        return self.count_params() * 4
```

Comment:
Is this because `shiftconv` needs to process 4 rotated input at once? If not, maybe you don't need to multiply by 4?

Reply:
This is to get the number of bytes